dyninstAPI/src/debug.h

```diff
@@ -178,7 +168,4 @@ DECLARE_PRINTF_FUNC(stackmods_printf_int);
 // And initialization
 extern bool init_debug();
 
-extern bool init_stats();
-extern bool print_stats();
```

Comment:
If you remove these declarations because they are unused, should the definitions in debug.C be removed as well?

Reply:
Oops. That's a rebase error. I'll add them back. Although they aren't actually used, but that's for another PR.